### PR TITLE
Dev alpha

### PR DESCRIPTION
**Pull request by devinleighsmith**
**REVIEWERS PLEASE READ**

This PR includes merging most recent dev into pims-alpha. Please filter the commits during your review to just:

3e51946
f11fa05

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -79,12 +77,14 @@
     "@types/node": "12.12.17",
     "@types/pretty": "2.0.0",
     "@types/react": "16.9.16",
+    "@types/react-bootstrap-typeahead": "^3.4.6",
```
Comment (Fosol) [inline frontend/package.json:80]: Specify version

File: frontend/src/constants/propertySteps.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Land steps enum, provides a list of steps corresponding to submittig bare land.
+ */
+export enum LandSteps {
```
Comment (Fosol) [inline frontend/src/constants/propertySteps.ts:4]: Are you going to have another enum for when associated land will only be 2 steps?

File: frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+        )}
+        {formikProps.dirty && stepper.current === 3 && (
+          <Button type="submit">Submit to Inventory</Button>
+        )}
+      </FormFooter>
+    </FormContentWrapper>
+  );
+};
+
+interface IBuildingForm {
```
Comment (Fosol) [inline frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx:140]: Should we split this file?

**Review by jdtoombs — APPROVED** [review]
Thank you for doing this!

File: frontend/src/features/mapSideBar/SidebarContents/SubmitPropertySelector.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        <Action onClick={addBuilding}>
+          <BuildingIcon className="svg" />
+          <ActionLabelWrapper>
+            <ActionPrimaryText>Add a building</ActionPrimaryText>
+          </ActionLabelWrapper>
+        </Action>
+        <Action onClick={addRawLand}>
+          <RawLandIcon className="svg" />
+          <ActionLabelWrapper>
+            <ActionPrimaryText>Add Raw Land</ActionPrimaryText>
```
Comment (Fosol) [inline frontend/src/features/mapSideBar/SidebarContents/SubmitPropertySelector.tsx:109]: Change to "Add Bare Land"

**Review by Fosol — CHANGES_REQUESTED** [review]
Your branch is `dev-alpha`.  Are you certain you want these changes in the `dev` branch?  I need to do a merge to master tomorrow morning for Sprint 17.

File: frontend/package.json (hunk 1 above)
Comment (devinleighsmith) [reply frontend/package.json:80]: Please only leave suggestions for the commits that I specified.

File: frontend/src/constants/propertySteps.ts (hunk 2 above)
Comment (jdtoombs) [reply frontend/src/constants/propertySteps.ts:4]: Just so Devin doesn't get the blame for any of my changes these are from my PR's

**Review by Fosol — APPROVED** [review]

**Comment by devinleighsmith** [comment]
Closing this because doing this as a rebase isn't going to work in this case. will re-open in a merge format.

**Pull request closed without merging**